docs/tools/cli/billing-group.rst

```diff
@@ -164,18 +157,18 @@ Deletes a billing-group
   * - Parameter
     - Information
   * - ``id``
-    - The ID of your billing group
+    - The billing group ID 
 
-**Example:** Delete your billing-group with id ``55b0e547-58f9-48de-8808-807d385d1f95``
+**Example:** Delete the billing group with id ``55b0e547-58f9-48de-8808-807d385d1f95``:
```

Comment:
`ID` should be capitalized like other places?

Reply:
Yes, good catch!